src/modules/image/providers/unsplash.ts

```diff
@@ -30,12 +31,21 @@ export class Unsplash {
   /**
    * Returns a random avatar url.
    *
+   * @see faker.internet.avatar
+   *
    * @example
    * faker.internet.avatar()
    * // 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/315.jpg'
+   *
+   * @deprecated
    */
-  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with unsplash.
   avatar(): string {
+    deprecated({
+      deprecated: 'faker.image.unsplash.avatar()',
+      proposed: 'faker.internet.avatar()',
+      since: 'v7.3.0',
+      until: 'v8.0.0',
```

Comment:
```suggestion
      since: '7.3',
      until: '8.0',
```

Changed in preparation #1067 .